subscription_test.go

```diff
@@ -146,7 +157,77 @@ var testCases = testCaseList{
 		expectation: &EventDeploymentInfo{
 			EventType: "deployment_info",
 			Timestamp: "2016-07-29T08:03:52.542Z",
-			Plan:      &DeploymentPlan{},
+			Plan:      &DeploymentPlan{
+				ID: "dcf63e4a-ef27-4816-e865-1730fcb26ac3",
+				Version: "2016-07-29T08:03:52.542Z",
+				Original: &Group{},
+				Target: &Group{},
+				Steps: []*DeploymentStep {
+					{
+						Action: "ScaleApplication",
+						App: "/my-app",
+					},
+				},
+			},
+			CurrentStep: &StepActions{
+				Actions: []struct {
+					Type string `json:"type"`
+					App  string `json:"app"`
+				}{
+					{
+						Type: "ScaleApplication",
+						App:  "/my-app",
+					},
+				},
+			},
+		},
+	},
+	// For marathon 1.X
```

Comment:
Can we have a similar comment for the older format that you added a test for above?